app/lib/forms/possible_funding.rb

```diff
@@ -14,11 +14,43 @@ def course
       @course ||= wizard.query_store.course
     end
 
+    def tsf_funding_amount
+      @tsf_funding_amount ||= if targeted_delivery_funding_eligibility?
+                                tsf_primary_plus_eligibility? ? 800 : 200
+                              end
```

Comment:
Looks like a weird indentation like this imo. I think this should work and look better inline

`@tsf_funding_amount ||= (targeted_delivery_funding_eligibility? && tsf_primary_plus_eligibility?) ? 800 : 200`

Reply:
Yeah i can take a look at this!